.github/workflows/php.yml

```diff
@@ -0,0 +1,64 @@
+name: CI FOSHttpCacheBundle
+
+env:
+  SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+
+on:
+  push:
+    branches: [ master, /^\d+\.\d+$/, hotfix/move-to-github-action ]
+  pull_request:
+    branches: [ master, /^\d+\.\d+$/, hotfix/move-to-github-action ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: ['7.2', '7.3']
+        composer-flag: ['--prefer-stable --prefer-lowest']
+        dependency: ['symfony/validator:^3.4.4 symfony/templating:^3.4.4', 'symfony/flex', 'symfony/lts:^3']
```

Comment:
i think this can also go away. to test the various php versions, we want to know if it works with default settings wihtout further restrictions. lts 3 is covered below, as is symfony/flex for symfony version 4.